libethereum/Executive.h

```diff
@@ -78,6 +78,21 @@ class StandardTrace
 	DebugOptions m_options;
 };
 
+
+struct AccountSnapshot
```

Comment:
The `AccountSnapshot` should be merged with the `Account` later on (at least tried to be merged). But renaming it to "AccountChanges" make no sense -- it can contain some account data, in out case it is used to keep "original" data of a changed account.